### ISSUE TITLE
(Firefox) Pullquote Does Not Work

### ISSUE THREAD

**Issue by riegelstamm** [uBlock-LLC/uBlock#664]
Neither the Pullquote.com extension nor bookmarklet work in Firefox. They work with Adblock Edge. Pullquote.com and blogads.com must be whitelisted in Adblock for it to work, so I whitelisted them in uBlock. However, the only way I can get Pullquote to work is to disable uBlock. I cannot find any other sites that might be preventing Pullquote from working.

I like the smaller RAM footprint of uBlock, so I hope I can get it to play nicely with Pullquote, which I use frequently.

**Comment by gorhill** [comment]
It looks to me it worked fine with uBlock from my side:

![d](https://cloud.githubusercontent.com/assets/585534/5994330/8112df88-aa3c-11e4-88e6-57d64c02f05b.png)

I will need repro steps, and details about versions, filter lists, custom filters, dynamic filtering rules, etc. Any information that might be useful in helping me reproduce.

**Comment by riegelstamm** [comment]
Hi Raymond,

Thanks for the quick response. Here are as many details as I can think of:

Mac OS X 10.9.5
Firefox 35.0.1 with only uBlock extension installed
only EasyList filter
Settings:

No custom filters
No custom rules
Whitelist: blogads.com, pullquote.com

JavaScript IS enabled.

Reprosteps:
1. Click Pullquote bookmarklet
2. Nothing happens
3. Disable uBlock & refresh page
4. Click Pullquote bookmarklet
5. "You can now Pullquote from this page" appears

Please let me know if I can provide anything else.

Keith

On Feb 1, 2015, at 5:03 PM, Raymond Hill notifications@github.com wrote:

It looks to me it worked finr with uBlock from my side:

I will need repro steps, and details about versions, filter lists, custom filters, dynamic filtering rules, etc. Any information that might be useful in helping me reproduce.

**Comment by gorhill** [comment]
> Click Pullquote bookmarklet

What is this? How does it look? (You can drag and drop an image in your comment, it will be automatically uploaded).

Is this what you are talking about:

![c](https://cloud.githubusercontent.com/assets/585534/5994643/f5f2f9e8-aa49-11e4-98d6-857677acbf59.png)

If so, I get that broken icon even when I start the browser without uBlock. Whether I enable/disable uBlock  that icon is always like this.

**Comment by riegelstamm** [comment]
It's on the Pullquote website: http://pullquote.com/bookmarklet
javascript:(function(){d=document;s=d.createElement('script');s.src='http://i.blogads.com/static/copystat/js/bookmarklet.min.js';d.getElementsByTagName('head')[0].appendChild(s);})();

On Feb 1, 2015, at 6:36 PM, Raymond Hill notifications@github.com wrote:

Click Pullquote bookmarklet

What is this? How does it look? (You can drag and drop an image in your comment, it will be automatically uploaded).

**Comment by gorhill** [comment]
Ok, I thought it was an add-on. Whitelisting's purpose is to disable uBlock for a whole site. What you want is different, you want to allow `i.blogads.com` everywhere. So it's a filter exception you need, not a whitelist directive. Here:

```
@@||i.blogads.com^
```

In "My filters" tab in the dashboard.

**Comment by riegelstamm** [comment]
I pasted that in to My filters, but still no go. I also tried it without the single quotes. What did I do wrong?

On Feb 1, 2015, at 7:37 PM, Raymond Hill notifications@github.com wrote:

Ok, I thought it was an add-on. Whitelisting's purpose is to disable uBlock for a whole site. What you want is different, you want to allow i.blogads.com everywhere. So it's a filter exception you need, not a whitelist directive. Here:

`@@||i.blogads.com^`
In "My filters" tab in the dashboard.

**Comment by gorhill** [comment]
> What did I do wrong?

It worked from here. Do all the steps carefully. Paste the filter as said (I removed the quotes afterward, my mistake). Click _Apply changes_. Ensure _My filters_ is checked in _3rd-party filters_. It worked here, so it has to work on your side. I can't debug for you remotely, you are the best placed to make it work. You also need to be more detailed. "Still no go" can mean a lot of things. I could quote and the Pullquote widget was showing, but I was asked to accept to run Flash which I refused. I am assuming it would have worked.

**Comment by riegelstamm** [comment]
The key was "Ensure My filters is checked in 3rd-party filters." After I did that, it worked. Thank you! I suggest including more instructions in the wiki. Will write a positive review shortly.

On Feb 1, 2015, at 10:46 PM, Raymond Hill notifications@github.com wrote:

What did I do wrong?

It worked from here. Do all the steps carefully. Paste the filter as said (I removed the quotes afterward, my mistake). Click Apply changes. Ensure My filters is checked in 3rd-party filters. It worked here, so it has to work on your side. I can't debug for you remotely, you are the best place to make it work. You also need to be more detailed. "Still no go" can means a lot of things. I could quote and the Pullquote widget was showing, but I was asked to accept to run Flash which I refuses. I am assuming it would have worked.
